mscore/scoreview.cpp

```diff
@@ -3302,8 +3302,13 @@ void ScoreView::addSlur()
       InputState& is = _score->inputState();
       if (noteEntryMode() && is.slur()) {
             const std::vector<SpannerSegment*>& el = is.slur()->spannerSegments();
-            if (!el.empty())
+            if (!el.empty()) {
                   el.front()->setSelected(false);
+                  // Now make sure that the slur segment is redrawn so that it does not *look* selected
```

Comment:
Wouldn't just [`update()`](https://doc.qt.io/qt-5/qwidget.html#update) call be sufficient to trigger redrawing? It doesn't look like re-layout is necessary here.

Reply:
You are right. Calling `update()` is sufficient. I have just made this change to the PR.